common/service/dynamicconfig/constants.go

```diff
@@ -827,6 +830,12 @@ const (
 	WorkerThrottledLogRPS
 	// ScannerPersistenceMaxQPS is the maximum rate of persistence calls from worker.Scanner
 	ScannerPersistenceMaxQPS
+	// ScannerMaxOrphanTasks is the maximum number of orphans to delete in one batch
+	ScannerMaxOrphanTasks
+	// ScannerTaskBatchSize is the number of tasks to query per call in the scavenger
+	ScannerTaskBatchSize
+	// ScannerMaxTasksPerJob is the number of tasks to process before letting another task list be processed
+	ScannerMaxTasksPerJob
```

Comment:
I am still going to understand what exactly the configs are. But so far, there is a gap between the name and the comment. Do you think you have better names for them? The names are important for configuration because many people don't really read description carefully.

Reply:
Here I just exposed variables that already existed having hardcoded values. I'm open to suggestions on better naming.

Reply:
Looks like some names are from existing code. It's okay if the names are not perfect in code. But when we expose as contract for users, we want to improve it. 
Just some ideas to rename:

`ScannerMaxOrphanTasks`=>ScannerGetOrphanTasksPageSize`

Reply:
e.g. `ScannerMaxTasksPerJob`=>`ScannerMaxTasksProcessedPerTasklistJob`

Reply:
e.g. `ScannerTaskBatchSize` => `ScannerBatchSizeForCompletingExpiredTasks`

Reply:
Also can you update comment/descripiton for`ScannerBatchSizeForCompleteTasksLessThanAckLevel`: 
```
The scavenger rely on it to decide if a tasklist can be deleted. So it's better to keep it a relatively high number to let be more efficient.